src/pages/Facility/services/inventory/externalSupply/deliveryOrder/SmartExternalDeliveryRow.tsx

```
@@ -0,0 +1,736 @@
+                  getItemFieldPath(index, "is_tax_inclusive"),
+                  !!checked,
+                );
+                markAsEdited();
+              }}
+              disabled={!productKnowledge}
+              className="h-3.5 w-3.5"
+            />
+            <span className="text-[10px] text-gray-500 whitespace-nowrap">
+              {t("tax_inclusive")}
+            </span>
+          </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)